flexmeasures/ui/templates/crud/users.html

```diff
@@ -34,49 +50,133 @@ <h3>All {% if not include_inactive %}active {% endif %}users</h3>
                 <th class="d-none">URL</th>
               </tr>
             </thead>
-            <tbody>
-              {% for user in users %}
-              <tr>
-                <td>
-                    {{ user.username }}
-                </td>
-                <td>
-                    <a href="mailto:{{ user.email }}" title="Mail this user">{{ user.email }}</a>
-                </td>
-                <td>
-                    {% for role in user.flexmeasures_roles %}
-                    {{ role.name }}{{ "," if not loop.last }}
-                    {% endfor %}
-                </td>
-                <td>
-                    <a href="/accounts/{{user.account.id}}" title="View accounts">{{ user.account.name }}</a>
-                </td>
-                <td>
-                    {{ user.timezone }}
-                </td>
-                <td title="{{  user.last_login_at | localized_datetime }}" data-sort="{{ user.last_login_at }}">
-                    {{ user.last_login_at | naturalized_datetime}}
-                </td>
-                <td title="{{  user.last_seen_at | localized_datetime }}" data-sort="{{ user.last_seen_at }}">
-                    {{ user.last_seen_at | naturalized_datetime}}
-                </td>
-                <td>
-                    {{ user.active }}
-                </td>
-                <td class="d-none">
-                    /users/{{ user.id }}
-                </td>
-              </tr>
-              {% endfor %}
+            <tbody id="users-table-body">
+              <!-- Dynamic rows will be rendered here -->
             </tbody>
           </table>
         </div>
+        <nav aria-label="Page navigation example">
+          <ul class="pagination" id="pagination-controls">
+            <!-- Pagination buttons will be rendred here -->
+          </ul>
+        </nav>
       </div>
     </div>
-    <div class="col-md-2"></div>
   </div>
 </div>
 
-{% block paginate_tables_script %} {{ super() }} {% endblock %}
+<script>
+  const basePath = window.location.origin;
+  const apiUrl = basePath + "/api/v3_0/users";
+  const tableBody = document.getElementById("users-table-body");
+  const paginationControls = document.getElementById("pagination-controls");
+  let currentPage = 1;
+
+  async function fetchUsers(page = 1) {
```

Comment:
This function and `renderTable` are repeated in accounts.html

We should put these somewhere central. Maybe `ui/static/js/tables-pagination.js`

But first: why not do it like the asset pagination PR did it? Look at `ui/crud/templates/assets.html` - that approach seems shorter. There, the `dataTable()` library is also used, which in general is an approach we take as it has some nice features.

Reply:
I'll work on that.